README.md

```diff
@@ -40,6 +40,31 @@ Note that launcher can also be compiled for other platforms, just need to link w
 bin/raylib-lua-sol examples/core_basic_window.lua
 ```
 
+## Prebuild On Windows
```

Comment:
Thank you so much for writing the Windows documentation!!!! 👍

Reply:
Thank you for your reply. No thanks.